Src/BootCamp.Chapter/Program.cs

```diff
@@ -2,11 +2,95 @@
 
 namespace BootCamp.Chapter
 {
-    class Program
+    public class Program
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            ConvertFromDecimalToBinary();
+            ConvertFromBinaryToDecimal();
+            PrintEmicon();
+        }
+
+        private static void PrintEmicon()
+        {
+            Console.OutputEncoding = System.Text.Encoding.Unicode;
+            Console.WriteLine("Give a a W, A, S or D");
+            var input = Console.ReadLine().ToUpper();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4977576)